docker/README.md

```diff
@@ -25,4 +24,119 @@ the table above):
 docker pull osgeo/grass-gis:<tag>
 ```
 
+## Provided images
+
+Choose the image and running method that best suits your needs based on
+the required features, available disk space, and whether you need GUI support.
+
+### Alpine-based image
+
+Image size: around 240 MB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-alpine
+```
+
+**Running:**
+
+```bash
+docker run -it --rm osgeo/grass-gis:releasebranch_8_4-alpine grass -c EPSG:4326 /tmp/grassproject
+```
+
+### Debian-based image
+
+Image size: around 1.20 GB
+
+**Installation:**
+
+```bash
+docker pull osgeo/grass-gis:releasebranch_8_4-debian
+```
+
+**Running:**
+
+```bash
+docker run -it --rm osgeo/grass-gis:releasebranch_8_4-debian grass -c EPSG:4326 /tmp/grassproject
+```
```

Comment:
This is the same for every image except for the tag. Too much stuff to read and maintain. Is there an example how other projects are addressing that?

Reply:
https://github.com/OSGeo/gdal/tree/master/docker